zulipterminal/ui_tools/boxes.py

```diff
@@ -728,6 +729,19 @@ def need_recipient_header(self) -> bool:
         else:
             raise RuntimeError("Invalid message type")
 
+    def _set_topic_links(self) -> None:
```

Comment:
Do you expect to expand upon this in future or call it in more than one place? If not then right now this appears to be a dict comprehension, so I think we can have it inline at the one call site?